Library/Formula/purescript.rb

```diff
@@ -0,0 +1,21 @@
+require "language/haskell"
+
+class Purescript < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.purescript.org"
+  url "http://hackage.haskell.org/package/purescript-0.6.8/purescript-0.6.8.tar.gz"
+  sha1 "70fd4d3109d61c34c8898a30d222c4b1ad8fd7a5"
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc"
+
+  def install
+    install_cabal_package
+  end
+
+  test do
+    # TODO:  figure out how to make use of the included cabal test suite
+    system "psc", "--version"
```

Comment:
We don't need to use the test suite but it would be good to do something slightly more involved.